packages/tracing/src/transaction.ts

```diff
@@ -142,7 +142,8 @@ export class Transaction extends SpanClass implements TransactionInterface {
       transaction.measurements = this._measurements;
     }
 
-    logger.log(`[Tracing] Finishing ${this.op} transaction: ${this.name}.`);
+    const op = this.op ? `${this.op} ` : '';
+    logger.log(`[Tracing] Finishing ${op}transaction: ${this.name}.`);
```

Comment:
They're jammed together because the const `op` includes the space if there's a value there, but not if there isn't, so that in cases where there's no value, you don't get a double space between "finishing" and "transaction".